.github/workflows/release.yml

```diff
@@ -65,3 +68,53 @@ jobs:
           done
         done
         hub release create "${ASSET_FLAGS[@]}" -F ${GITHUB_WORKSPACE}/release-note.txt --draft "${RELEASE_TAG}"
+  
+  containerize:
+    runs-on: ubuntu-20.04
+    name: Containerize
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ env.GHCR_SLUG }}
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=ref,event=pr
+            type=edge
```

Comment:
Needs comments why we need them. I think having only `vX.X.X` tag is just enough.